src/test/java/nl/knaw/dans/ingest/core/service/mapper/DepositToDvDatasetMetadataMapperTest.java

```diff
@@ -87,6 +90,38 @@ void to_dataverse_dataset() throws Exception {
             .writeValueAsString(result);
     }
 
+    @Test
+    void toDataverseDataset_should_include_otherId() throws Exception {
+        var mapper = getMapper();
+        var doc = readDocument("dataset-simple.xml");
+
+        var vaultMetadata = new VaultMetadata("pid", "bagId", "nbn", "doi:a/b", "otherIdVersion", "swordToken");
+
+        var result = mapper.toDataverseDataset(doc, null, null, null, null, vaultMetadata);
+        var str = new ObjectMapper()
+            .writer()
+            .withDefaultPrettyPrinter()
+            .writeValueAsString(result);
+
+        assertTrue(str.contains("doi:a/b"));
```

Comment:
```suggestion
        assertThat(str).contains("doi:a/b");
```

results in `Expecting actual: ... to contain: "..." `
then you don't have to debug to figure out `...`